apps/extensions/src/content/panel/components/containers/dashboardLayout/elements/SidebarMenuItems/SidebarMenuItem.tsx

```
@@ -17,21 +20,126 @@ const SidebarMenuItem: React.FC<ISidebarMenuItemProps> = ({
+              >
+                <span onClick={onClick} className="tw-flex">
+                  <div className={classNames(styles.icon)}>{icon}</div>
+                  <div className={classNames(styles.title)}>{title}</div>
+                </span>
+                <div
+                  className="tw-absolute tw-right-4 "
+                  onClick={(e) => {
+                    e.preventDefault();
+                    handleToggle(e);
+                  }}
+                >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)